### ISSUE TITLE
Error summaries should only be programatically focusable, not keyboard focusable

### ISSUE THREAD

**Issue by edwardhorsford** [nhsuk/nhsuk-frontend#944]
# Bug Report

The [GOV.UK Error summary](https://design-system.service.gov.uk/components/error-summary/) is only focusable programatically - JS is used to focus it on page load.

The [NHS Error summary](https://service-manual.nhs.uk/design-system/components/error-summary) is different in that it's currently keyboard focusable. From a conversation on NHS slack it sounds like this is because the GOV.UK one has had an update since the NHS one was produced.

## What is the issue?

I don't think there's any reason for the error summary to be keyboard focusable - and it's probably good to mirror the GOV.UK one.

**Comment by andymantell** [comment]
For reference, it got added to govuk in January 2022:
https://github.com/alphagov/govuk-frontend/commit/e1b17e0af8da11d9db72cbd99020444d69d9c4cc

Which was after our initial port of the JS:
https://github.com/nhsuk/nhsuk-frontend/commit/268e164609d0452b2bcca076b1c21740a69a7177

**Comment by colinrotherham** [comment]
This will be fixed by https://github.com/nhsuk/nhsuk-frontend/issues/1035 as part of v10.1.0

It's almost a duplicate but this one is more for `tabindex` rather than the alert role changes

**Comment by colinrotherham** [comment]
Closing as fixed in https://github.com/nhsuk/nhsuk-frontend/issues/1035
